python/ray/tune/tests/tutorial.py

```diff
@@ -105,7 +105,7 @@ def train_mnist(config):
         acc = test(model, test_loader)
 
         # Send the current training result back to Tune
-        tune.report(mean_accuracy=acc)
+        session.report(mean_accuracy=acc)
```

Comment:
Will need to also include the proper import at the top: 

`from ray.air import session`